src/igvfd/types/sample.py

```diff
@@ -753,11 +754,7 @@ class WholeOrganism(Biosample):
     item_type = 'whole_organism'
     schema = load_schema('igvfd:schemas/whole_organism.json')
     embedded_with_frame = Biosample.embedded_with_frame
-    audit_inherit = Biosample.audit_inherit + [
-        'disease_terms',
-        'treatments',
-        'modifications',
-    ]
```

Comment:
Not sure why but Whole organisms audit inheritance is redundant with Biosample audit inheritance.